internal/api/api_test.go

```diff
@@ -0,0 +1,386 @@
+// Copyright (c) 2020-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+package api
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/gorilla/mux"
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	mock_api "github.com/mattermost/awat/internal/mocks/api"
+	"github.com/mattermost/awat/model"
+)
+
+func TestTranslations(t *testing.T) {
+	logger := MakeLogger(t)
+	mockController := gomock.NewController(t)
+	store := mock_api.NewMockStore(mockController)
```

Comment:
Is there a script / Makefile rule that generates the mock?

I might be missing something but I am 4/5 for adding it. It can be a pain when someone not familiar with the project needs to jump in and make some changes to interfaces.

Reply:
No there is not and it's absolutely an oversight. You are totally correct and I will add a Makefile target for it.